_inc/client/plans/single-product-backup.jsx

```diff
@@ -155,15 +186,93 @@ function PlanRadioButton( {
 	);
 }
 
-// eslint-disable-next-line no-unused-vars
+function PlanSavingsText( { selectedBackup, billingTimeFrame, currencyCode } ) {
+	if ( ! selectedBackup ) {
+		return null;
+	}
+	const savingsCurrencyObject = getCurrencyObject( selectedBackup.potentialSavings, currencyCode );
+	const savings = formatCurrency( savingsCurrencyObject );
+
+	// TODO: Provide way to switch billing time frame state.
+	const switchComponent = <a />;
+
+	let promptText;
+	if ( 'yearly' === billingTimeFrame ) {
+		promptText = __( 'You are saving %(savings)s by paying yearly. {{a}}Switch to monthly{{/a}}', {
+			args: { savings },
+			components: {
+				a: switchComponent,
+			},
+		} );
+	} else if ( 'monthly' === billingTimeFrame ) {
+		promptText = __(
+			'You could be saving %(savings)s by paying yearly. {{a}}Switch to yearly{{/a}}',
+			{
+				args: { savings },
+				components: {
+					a: switchComponent,
+				},
+			}
+		);
+	}
+
+	if ( ! promptText ) {
+		return null;
+	}
+	return (
+		<p>
+			<em>{ promptText }</em>
+		</p>
+	);
+}
+
+function UpgradeButton( { selectedUpgrade, billingTimeFrame, currencyCode, onClickHandler } ) {
+	if ( ! selectedUpgrade ) {
+		return null;
+	}
+	const { link, name, fullPrice, discountedPrice, type } = selectedUpgrade;
+	let billingTimeFrameString = '';
+	if ( 'yearly' === billingTimeFrame ) {
+		billingTimeFrameString = __( 'per year', { context: 'Amount of money per time unit.' } );
+	} else if ( 'monthly' === billingTimeFrame ) {
+		billingTimeFrameString = __( 'per month', { context: 'Amount of money per time unit.' } );
+	}
+
+	const currencyObject = getCurrencyObject( discountedPrice || fullPrice, currencyCode );
+
+	return (
+		<div className="single-product-backup__upgrade-button-container">
+			<Button href={ link } onClick={ onClickHandler( type ) } primary>
+				{ __( 'Upgrade to %(name)s for %(price)s/%(billingTime)s', {
```

Comment:
Even though the designs show the slash (`/`) here, I believe it's redundant. The time frame copy already states it is "per [time period]".

Reply:
Done in ae5bde1c126081336373f935b81bc3269b94bee2!